README.md

```diff
@@ -1 +1,9 @@
-# etcd-rpm
\ No newline at end of file
+# etcd-rpm
+
+## Project Description
+
+This repository provides a simple solution for packaging etcd as an RPM package using nfpm. With the help of GitHub Actions, the RPM package is automatically built and published as a release artifact.
+
```

Comment:
better to add description for `how to release a new version`

Reply:
updated.